enterprise/server/remote_execution/containers/docker/docker.go

```diff
@@ -25,12 +25,14 @@ import (
 	dockertypes "github.com/docker/docker/api/types"
 	dockercontainer "github.com/docker/docker/api/types/container"
 	dockerclient "github.com/docker/docker/client"
+	units "github.com/docker/go-units"
```

Comment:
nit: prefix import alias with `docker` (for consistency with above imports)

Reply:
this library is called `go-units` it just happens to be in the docker organization: https://github.com/docker/go-units

Reply:
woops -- units sg